Chapter-3-microservice-extraction/Contracts/Src/Fitnet.Contracts.Api.UnitTests/Fitnet.Contracts.Api.UnitTests.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Have you rebased your version based on current one? Then Please do not forget to remove Contracts folder from Root and ModularMonolith (as they do not exist anymore - were both renamed)